rgbd/src/client_shm.cpp

```diff
@@ -148,7 +148,7 @@ ImagePtr ClientSHM::nextImage()
     if (nextImage(*img))
         return img;
     else
-        return ImagePtr();
+        return nullptr;
```

Comment:
Update the docstring to define this new behaviour.

Reply:
This is the same behaviour. It is just easier to not hardcode the type here, might the type change in the future. Plus more consistent.